stdlib/public/SDK/Foundation/NSError.swift

```diff
@@ -406,7 +406,15 @@ extension _BridgedNSError where Self.RawValue: FixedWidthInteger {
     self.init(rawValue: RawValue(_bridgedNSError.code))
   }
 
+  public func hash(into hasher: inout Hasher) {
+    hasher.combine(_code)
+  }
+
   public var hashValue: Int { return _code }
```

Comment:
Should you remove `hashValue` and let it pick up the default implementation?

Reply:
That's a good point; I don't see any reason why we wouldn't want that. (In fact, the same applies to `_rawHashValue`.) I'll remove these.